src/config/Config.zig

```diff
@@ -4036,7 +4036,7 @@ pub const Palette = struct {
         const eqlIdx = std.mem.indexOf(u8, value, "=") orelse
             return error.InvalidValue;
 
-        const key = try std.fmt.parseInt(u8, value[0..eqlIdx], 10);
+        const key = try std.fmt.parseInt(u8, value[0..eqlIdx]);
```

Comment:
We use Zig 0.13 so the final arg has to be `0` to imply the base based on prefix.